ocs_ci/framework/__init__.py

```diff
@@ -77,22 +77,23 @@ def merge_dict(orig: dict, new: dict) -> dict:
         new  (dict): The object which is the source of the update
 
     Example:
-    orig = {
-        'dict': {'one': 1, 'two': 2},
-        'list': [1, 2],
-        'string': 's',
-    }
-    new = {
-        'dict': {'one': 'one', 'three': 3},
-        'list': [0],
-        'string': 'x',
-    }
-    merge_dict(orig, new) ->
-    {
-        'dict': {'one': 'one', 'two': 2, 'three': 3}
-        'list': [0],
-        'string', 'x',
-    }
+        orig = {\
```

Comment:
is the \ a requirement for sphinx?

Reply:
I'm curious as well, I see they've been added in a number of places.

Reply:
@vasukulkarni @clacroix12 I think, it is because sphinx doesn't like the various indentation in the example block... @petr-balogh but wouldn't be better to use some "code snippet" block (called [Literal Blocks](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks))?
Maybe it might be enough to add second colon after the Example (like `Example::`) and one empty line.

Reply:
Yes, it was cause sphinx complained cause of indentions: :`Definition list ends without a blank line; unexpected unindent.`

I tried to google solution and found one with backslashes but also didn't like it much.

@dahorak thanks, that works better with literal blocks ;) will change